helpers/src/db/pagination.rs

```diff
@@ -7,6 +7,19 @@ use diesel::QueryId;
 
 const DEFAULT_PER_PAGE: i64 = 10;
 
+/**
+This module provides pagination for database queries.
+
+```
+<schema-module>::table
+    .into_boxed()
+    .paginate(<page>)
+    .per_page(<records-per-page>)
+    .load_and_count_pages(<database-connection>)
+```
+This returns a tuple `(Vec<T>, i64)` with the records of the requested page as the first and the amount of pages in the second element.
+**/
```

Comment:
I guess this style of comment won't be rendered as doc-comment?

Reply:
When I do `cargo doc` I can see it in the module documentation. Do you mean something else?

Reply:
Ah okay. Just looked it up.
* no doc-comment
    ```rust
    /* */
    ```
* doc-comment
    ```rust
    /** **/
    ```